data/test/standalone-topography/rainbow.yaml

```diff
@@ -0,0 +1,36 @@
+structures:
+  - name: stripe
+    structure:
+      palette:
+        'r': "#ff0000"
+        'o': "#ff8800"
+        'y': "#ffff00"
+        'g': "#00ff00"
+        'b': "#00ffff"
+        'i': "#0000ff"
+        'v': "#ff00ff"
+      map: |
+        roygbiv
+mask: '.'
+palette:
+  'r':
+    structure:
+      name: stripe
+    cell: "#ffffff"
```

Comment:
What does the `cell: "#ffffff"` mean?

Reply:
In this file, the type of the cells are hexadecimal color codes.  Generically, for "structure placement" palette entries, it is mandatory to specify a default value for the cell in the case that the upper-left coordinate of the structure is itself transparent.  In this case I arbitrarily chose white as the default.